### PR TITLE
Update CAPI for the v2 use

### PR DESCRIPTION
**Pull request by dew326**
Update CAPI to meet changes in v2

### CODE REVIEW

**Comment by andrerom** [comment]
how was token and siteaccess handled before? would have thought it was injected somehow more cleanly or?

**Comment by lserwatka** [comment]
Yes, it should CSRF token gathering be moved outside of CAPI.

File: src/ConnectionManager.js [hunk 1]
```diff
@@ -114,9 +116,18 @@ define(["structures/Response", "structures/Request", "structures/CAPIError"],
             headers['X-Siteaccess'] = this._siteAccess;
         }
 
+        requestUrl = this._endPointUrl + url;
+
+        if (this._siteAccess && this._token) {
+            headers['X-Siteaccess'] = this._siteAccess;
+            headers['X-CSRF-Token'] = this._token;
+
+            requestUrl = url;
```
Comment (lserwatka) [inline src/ConnectionManager.js]: If you don't need `this._endPointUrl` then we should rather pass an empty string from outside of this function.

**Comment by lserwatka** [comment]
+1 if travis agrees.

**Pull request merged**